src/components/SeriesNav/SeriesNav.js

```diff
@@ -47,8 +50,16 @@ function SeriesNav({
   context,
   PayNote,
   onEpisodeClick,
-  aboveTheFold
+  aboveTheFold,
+  seriesDescription
 }) {
+  const showSeriesDescripion =
+    seriesDescription !== undefined
+      ? seriesDescription
+      : inline
+      ? true
+      : undefined
```

Comment:
Fancy way of writing with [`Nullish coalescing operator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):
```suggestion
    seriesDescription ?? (
      inline
      ? true
      : undefined
    )
```